src/meta/src/hummock/error.rs

```diff
@@ -99,15 +100,11 @@ impl From<Error> for risingwave_common::error::RwError {
     }
 }
 
-// TODO: as a workaround before refactoring `MetadataModel` error
-impl From<risingwave_common::error::RwError> for Error {
-    fn from(error: RwError) -> Self {
-        match error.inner() {
-            ErrorCode::InternalError(err) => Error::InternalError(err.to_owned()),
-            ErrorCode::ItemNotFound(err) => Error::InternalError(err.to_owned()),
-            _ => {
-                panic!("conversion not supported");
-            }
+impl From<MetadataModelError> for Error {
+    fn from(err: MetadataModelError) -> Self {
+        match err {
+            MetadataModelError::MetaStoreError(e) => e.into(),
```

Comment:
Will hummock manager directly yield `MetaStoreError`? If not, I suggest replacing it with `MetadataModelError` and directly wrap an error in that variant.

Reply:
Let us do it later when we replace `RwError` to meta's own error.